db/config.cc

```diff
@@ -747,6 +747,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "Enable or disable the native transport server. Uses the same address as the rpc_address, but the port is different from the rpc_port. See native_transport_port.")
     , native_transport_port(this, "native_transport_port", "cql_port", value_status::Used, 9042,
         "Port on which the CQL native transport listens for clients.")
+    , maintenance_socket(this, "maintenance_socket", value_status::Used, "/tmp/scylla/cql.maintenance",
+        "The Unix Domain Socket a node uses for maintenance port. If set to 'ignore', the node will not start the maintenance port")
```

Comment:
Should have a brief mention about this being a full-permission thing.

Reply:
I think the default should be that it is disabled. And we should maybe make this live-update and add a REST API to enable/disable it.

Reply:
> I think the default should be that it is disabled. 

Why? IIRC @avikivity had the opposite idea.

Reply:
Ok, I don't have a strong opinion.